src/main/c/seasocks/ZlibContext.cpp

```diff
@@ -86,17 +86,27 @@ struct ZlibContext::Impl {
         deflateStream.next_in = (unsigned char*) input;
         deflateStream.avail_in = inputLen;
 
-        do {
+        if (inputLen > 0) do {
```

Comment:
This formatting is a bit confusing (at least for me). Especially "closing" the `if` with a `while`.